net/net.sh

```diff
@@ -52,6 +52,7 @@ Operate a configured testnet
 
    --hashes-per-tick NUM_HASHES|sleep|auto
                                       - Override the default --hashes-per-tick for the cluster
+   -n NUM_NODES                       - Number of nodes to apply command to.
```

Comment:
nit: the additional context of *full* would probably be helpful:
```
-n NUM_FULL_NODES                       - Number of full nodes to apply command to.
```

Reply:
Added that. thanks.